sql/parser/normalize.go

```diff
@@ -399,7 +399,7 @@ func (v *isConstVisitor) Visit(expr Expr, pre bool) (Visitor, Expr) {
 // (i.e. it does not contain a VariableExpr, QualifiedName, etc).
 func isConst(expr Expr) bool {
 	v := isConstVisitor{isConst: true}
-	expr = WalkExpr(&v, expr)
+	_ = WalkExpr(&v, expr)
```

Comment:
no, that was a judgement call. I preferred to keep the indication that the function returns a value.